src/globals.css

```diff
@@ -486,3 +486,11 @@ div[id^="headlessui-menu-items"] {
   font-size: 13px;
   padding: 7px 14px;
 }
+
+.learn-subtitle {
+  @apply mt-2;
+  @apply pb-2;
+  @apply border-neutral-200;
+  @apply border-b;
```

Comment:
border color should be improved on dark mode 
<img width="1323" alt="image" src="https://github.com/user-attachments/assets/f3894f55-4a16-4ecc-a8d7-d803d0bfff66">